### PR TITLE
test: fix TC247908 and TC247996

### LINKED ISSUE #1108: Fix TC247908
## Description ##
"TC247908 Local copybooks - Delete watched folders" needs to be updated.

### PR DESCRIPTION
**Pull request by nalmabrcom**
Fixes #1108 

```

  (Run Finished)


       Spec                                              Tests  Passing  Failing  Pending  Skipped
  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ ✔  LSP/F96117.spec.js                       01:18        2        2        -        -        - │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘
    ✔  All specs passed!                        01:18        2        2        -        -        -

```

Signed-off-by: Maryna Nalbandian <maryna.nalbandian@broadcom.com>

### CODE REVIEW

File: tests/cypress/integration/LSP/F96117.spec.js [hunk 1]
```diff
@@ -158,39 +159,38 @@ context('This is a F96117 spec', () => {
...
-        .type('{backspace}')
-        .type('{backspace}')
-        .type('{backspace}')
+        .type('{backspace}{backspace}{backspace}')
         .type('CA. ', { delay: 100 })
         .type('{ctrl}{c}')
         .closeCurrentTab();
       cy.get('.theia-button.main').click();
 
       // Open file and check that vriable is not available
```
Comment (temanbrcom) [inline tests/cypress/integration/LSP/F96117.spec.js:192]: "variable"

**Review by temanbrcom — APPROVED** [review]
🌵

File: tests/cypress/integration/LSP/F96117.spec.js [hunk 2]
```diff
@@ -18,6 +18,8 @@
 
 //F96117 - Support for Copybooks in Git/bridge for Git
 
+const backspace3times = '{backspace}{backspace}{backspace}';
```
Comment (grianbrcom) [inline tests/cypress/integration/LSP/F96117.spec.js:21]: You can do: `'{backspace}'.repeat(3)`.

**Review by grianbrcom — APPROVED** [review]
:flamingo:

**Review by grianbrcom — APPROVED** [review]
:shaved_ice:

**Pull request merged**